exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,96 @@
+    },
+    {
+      "description": "returns verse number 6",
+      "property": "verse",
+      "input": [6],
+      "expected": "On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 7",
+      "property": "verse",
+      "input": [7],
+      "expected": "On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 8",
+      "property": "verse",
+      "input": [8],
+      "expected": "On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 9",
+      "property": "verse",
+      "input": [9],
+      "expected": "On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 10",
+      "property": "verse",
+      "input": [10],
+      "expected": "On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 11",
+      "property": "verse",
+      "input": [11],
+      "expected": "On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "returns verse number 12",
+      "property": "verse",
+      "input": [12],
+      "expected": "On the twelfth day of Christmas my true love gave to me, twelve Drummers Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n"
+    },
+    {
+      "description": "finds multiple verses, which are a fragment of the song",
+      "property": "verses",
+      "input": [1, 3],
```

Comment:
There is no test for verses that doesn't start at 1
what should happen in that case?